Assets/Scripts/Posession.cs

```diff
@@ -22,6 +22,7 @@ private void OnTriggerStay(Collider other)
         if(other.gameObject.name == "DeadDude" && Input.GetKeyDown("space"))

```

Comment:
One problem here is that we don't change an Enemy's name to DeadDude when its health reaches zero. This would need to be done in Enemy.cs, in the `ChangeHealth` function. When an Enemy's health reaches 0, we should mark it as dead.

Reply:
Done thanks for the feedback